Error while checking build/O2/fullCI for e563022fb0a56e41a5ee48b4012482dcad5e9109 at 2023-08-03 12:41:
```
## sw/BUILD/O2-latest/log
/sw/SOURCES/O2/11695-slc8_x86-64/0/DataFormats/Detectors/TRD/src/T0FitHistos.cxx:22:6: error: no declaration matches 'void o2::trd::T0FitHistos::fill(const std::vector<o2::trd::PHData>&)'
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/11695/e563022fb0a56e41a5ee48b4012482dcad5e9109/build_O2_fullCI/pretty.html).

--- a/DataFormats/Detectors/TRD/include/DataFormatsTRD/T0FitHistos.h
+++ b/DataFormats/Detectors/TRD/include/DataFormatsTRD/T0FitHistos.h
@@ -39,7 +39,7 @@ class T0FitHistos
   auto getADC(int index) const { return mADC[index]; }
   auto getNEntries() const { return mNEntriesTot; }
 
-  void fill(const std::vector<o2::trd::PHData> data);
+  void fill(const std::vector<o2::trd::PHData>& data);
   void merge(const T0FitHistos* prev);
   void print();
 

